src/api/index.ts

```diff
@@ -1,26 +1,24 @@
 import { SyncStatePayload, SettingsState } from 'types'
 
-export const requestCategories = () =>
-  new Promise((resolve, reject) => {
-    const data = localStorage.getItem('categories') || '[]'
-
-    if (data) {
-      resolve(JSON.parse(data))
-    } else {
-      reject({ message: 'Something went wrong' })
-    }
-  })
+type IPromiseCallbackFun = (value?: any) => void;
+type IGetLocalStorage = (key: string, errorMsg?: string) => (resolve: IPromiseCallbackFun, reject: IPromiseCallbackFun) => void;
```

Comment:
Right now the Repo isn't using the I nomenclature.  Let's remove that as to try to keep standards in line